src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -49,6 +49,31 @@ public boolean connect(String database, String user, String password) {
         }
     }
 
+    public boolean insert(String tableName, DataSet dataset) {
+        String csvValues = csvValues(dataset);
+        String sqlQuery = String.format("INSERT INTO %s VALUES(%s)", tableName, csvValues);
+        try(Statement statement = connection.createStatement()){
+            statement.executeUpdate(sqlQuery);
+            return true;
+        } catch (SQLException e) {
+            return false;
+        }
+    }
+
+    private String csvValues(DataSet dataset) {
+        String[] values = dataset.values();
+        StringBuilder csvValues = new StringBuilder();
+        for (String value : values) {
+            csvValues.append("'")
+                    .append(value)
```

Comment:
Maybe use local variables middleDelimiter and endDelimiter or something similar?

Reply:
I'm not sure I understood correct your remark... I don't see any reason to add any local variables here and also not sure that I undestand where you want them to be. Could you please show the way you see that fragment of code